frontend/test/jest/utils.tsx

```diff
@@ -1,12 +1,31 @@
 import { render, RenderOptions, RenderResult } from '@testing-library/react';
-import { AppProvider } from '@ifixit/app';
+import { AppProviders } from '@components/common';
```

Comment:
What's the difference between `{ AppProvider } from '@ifixit/app'` and `{ AppProviders } from '@components/common'`?

Reply:
Oh my bad, just read the commit message.